src/pages/Assistance/Assistance.tsx

```diff
@@ -172,6 +172,10 @@ const Assistance = () => {
     };
   };
 
```

Comment:
You are preventing clipboard event nesting on the component's onPaste and onCopy, merge these methods into one using a talking name like preventClipboardEvents